From dc4a8fbb194cad1e040ce8b42de7a669d558251d Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Tue, 11 Sep 2018 03:33:26 +0000
Subject: [PATCH 241/249] Revert "nash: Link a custom TWRP image"

* As of 09-10-2018 we resolved all the issues we ran into on official
  TWRP builds on non-Pixel A/B devices.

** kDownloadPayloadPubKeyVerificationError - Resolved by:
   https://github.com/omnirom/android_system_update_engine/commit/ebdb97fc037e00db7e409db82ea5357f0a380f8c
   Signed builds wouldn't flash if they were signed with anything other
   than test-keys previous to this.

** kInstallDeviceError - Resolved by:
   Use the same fstab for OS/Recovery.
   This was happening because we relabeled /oem to /vendor in Lineage
   builds, whereas Stock and other ROM's kernel's didn't have it defined
   in the DT provided fstab.

** kPayloadTimestampError - Resolved by:
   As of 09-10-2018 TWRP's build servers pick in a change to always
   retirn 0 here.

* As a side note, all non-Pixel A/B devices should ask for a rebuild of
  their TWRP image/installer ASAP.

This reverts commit 75bee41cc4700917a7c39bc9dcddf8e96e0d4386.

Change-Id: Ibd1696ec9f2d11a5073989e4ec258702057cf0c2
---
 _data/devices/nash.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/_data/devices/nash.yml b/_data/devices/nash.yml
index 1c8eecb..005bf52 100644
--- a/_data/devices/nash.yml
+++ b/_data/devices/nash.yml
@@ -12,7 +12,6 @@ cpu: Kryo
 cpu_cores: '8'
 cpu_freq: 4 x 2.45 GHz + 4 x 1.9 GHz
 current_branch: 15.1
-custom_twrp_link: https://androidfilehost.com/?fid=3700668719832238233
 depth: 6.1 mm (0.24 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 540
-- 
2.17.1

